wren-ui/src/apollo/server/connectors/postgresConnector.ts

```diff
@@ -63,7 +64,15 @@ export class PostgresConnector
       return true;
     } catch (err) {
       logger.error(`Error connecting to Postgres: ${err}`);
-      return false;
+      if (err.code === 'ECONNREFUSED') {
+        throw Errors.create(Errors.GeneralErrorCodes.CONNECTION_REFUSED, {
+          originalError: err,
+        });
+      } else {
+        throw Errors.create(Errors.GeneralErrorCodes.CONNECTION_ERROR, {
+          originalError: err,
+        });
+      }
```

Comment:
maybe sth like following ?
```suggestion
     const errCode = (err.code === 'ECONNREFUSED') ? Errors.GeneralErrorCodes.CONNECTION_REFUSED : Errors.GeneralErrorCodes.CONNECTION_ERROR;
    throw Errors.create(errCode, {
        originalError: err,
      });
```